### ISSUE TITLE
Access to an underlying options object translated directly into API params

### ISSUE THREAD

**Issue by gatapia** [pbugnion/gmaps#151]
I love the simple abstraction you have here but support for more options would also be good. Something like the following:

```
geneva2zurich = gmaps.directions_layer(geneva, zurich)
geneva2zurich.additional_options['mode'] = 'bicycling'
geneva2zurich.additional_options['avoid'] = 'tolls'
```

This can just translate into query string params to the API.

**Comment by pbugnion** [comment]
Thanks for raising an issue! 

We definitely need additional functionality in the directions layer (and *jupyter-gmaps* as a whole). See issue #132 for instance. I'd much rather expose those options directly in Python, rather than have a dictionary of key-value pairs that gets given to the JavaScript layer. Ie. I'd rather add additional keyword arguments to the `directions_layer` function.

> This can just translate into query string params to the API.

I'm not sure what this means -- we rely on the JavaScript API, we don't talk to a REST API directly.

**Comment by gatapia** [comment]
Added PR #153

**Comment by pbugnion** [comment]
That's really great! Thanks for submitting a PR.

**Comment by pbugnion** [comment]
Addressed in part by #153 . I'll leave this issue open since there are additional parameters that aren't available yet (e.g. the travel mode).

See also issue #132 .

**Comment by pbugnion** [comment]
Travel mode addressed in issue #157. Closing this issue in favour of #132.
